ddtrace/llmobs/_evaluators/ragas/models.py

```diff
@@ -0,0 +1,35 @@
+import typing as t
```

Comment:
```suggestion
from typing import List
...
```
For consistency let's import from typing instead of using the shorthand form